<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manufacturer_dlg</class>
 <widget class="QDialog" name="manufacturer_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="box_notes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_notes">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>144</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_mfg_pn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_mfg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_manufacturer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Manufacturer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_manufacturer_pn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Manufacturer Part Number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_datasheet_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>52</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_rohs_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_ce_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_ul_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_datasheet">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>52</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_rohs">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_ce">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_ul">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_datasheet_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>35</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Datasheet Link</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rohs_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>73</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RoHS Conformance Link</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ce_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>113</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CE Conformance Link</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ul_link">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>153</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UL Conformance Link</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_datasheets">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Datasheet and Conformance Info</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_manufacturer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Manufacturer Info</string>
   </property>
   <widget class="QCheckBox" name="cb_preferred">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>80</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Preferred</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_deactivated">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_active">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>104</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
  </widget>
  <zorder>group_manufacturer</zorder>
  <zorder>group_datasheets</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>box_notes</zorder>
  <zorder>lbl_notes</zorder>
  <zorder>box_mfg_pn</zorder>
  <zorder>box_mfg</zorder>
  <zorder>lbl_manufacturer</zorder>
  <zorder>lbl_manufacturer_pn</zorder>
  <zorder>box_datasheet_link</zorder>
  <zorder>box_rohs_link</zorder>
  <zorder>box_ce_link</zorder>
  <zorder>box_ul_link</zorder>
  <zorder>btn_browse_datasheet</zorder>
  <zorder>btn_browse_rohs</zorder>
  <zorder>btn_browse_ce</zorder>
  <zorder>btn_browse_ul</zorder>
  <zorder>lbl_datasheet_link</zorder>
  <zorder>lbl_rohs_link</zorder>
  <zorder>lbl_ce_link</zorder>
  <zorder>lbl_ul_link</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
